examples/hooks-ssr/README.md

```diff
@@ -0,0 +1,16 @@
+This example shows how to do server side rendering with React InstantSearch.
+
+## Clone the example
+
+```
+curl https://codeload.github.com/algolia/react-instantsearch/tar.gz/master | tar -xz --strip=2 react-instantsearch-master/examples/hooks-ssr
+```
+
+## Start the example
+
+```sh
+yarn install --no-lockfile
+yarn start
+```
+
+Read more about React InstantSearch in the [documentation](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/react/).
```

Comment:
This is targeting the RIS DOM docs, we should link to the new RIS Hooks.

Until it's ready we should probably remove it to avoid any confusion.

Reply:
See be2ca9dbcca3cdd8044dc15a978b85b440b63be5.